mgnify_backlog/mgnify_handler.py

```diff
@@ -447,6 +450,10 @@ def update_annotation_jobs_from_accessions(self, run_or_assembly_accessions=None
             self.update_annotation_jobs_status(jobs, status_description)
             logging.info('Updated AnnotationJob status...')
 
+        if result_status:
+            self.update_annotation_jobs_result_status(jobs, result_status)
```

Comment:
The method `update_annotation_jobs_result_status` has only one line, IMO it doesn't make sense to have it.
Just do:
```python
if result_status:
    annotation_jobs.update(result_status=result_status)
    logging.info('Updated AnnotationJob result_status...')
```

Reply:
I've changed this function and also 2 functions with 1 line inside (set_priority and set_directory)

Reply:
brilliant :). Thanks